app/views/unattended/snippets/_remote_execution_ssh_keys.erb

```diff
@@ -0,0 +1,16 @@
+<%#
+  kind: snippet
+  name: remote_execution_ssh_keys
+%>
+
+<% unless @host.params['remote_execution_ssh_keys'].blank? %>
+<% ssh_user = @host.params['remote_execution_ssh_user'] || 'root' %>
+
+mkdir -m=0700 -p ~<%= ssh_user %>/.ssh
```

Comment:
Sure, makes sense.  What do you think about the snippet in general?

I don't like that we need the user to modify their provisioning template.  I was thinking of adding generic ssh key support to the community-templates repo, and using a more generic parameter name like ssh_keys.